src/libraries/System.IO.Compression.Brotli/src/System/IO/Compression/BrotliStream.cs

```diff
@@ -72,6 +82,13 @@ protected override void Dispose(bool disposing)
             }
         }
 
+        /// <summary>Asynchronously releases the unmanaged resources used by the <see cref="System.IO.Compression.BrotliStream" />.</summary>
+        /// <returns>A task that represents the asynchronous dispose operation.</returns>
+        /// <remarks><![CDATA[
```

Comment:
- The `## Remarks` header is excluded to save lines of code. The MS Docs website will show the Remarks header whether it's explicitly mentioned in the remarks comments or not.
    - Double newlines were collapsed into one also to save lines of code. The MS Docs website should be able to render paragraph line height correctly without the need for two line break characters.

Reply:
Could it be possible to avoid emitting `![CDATA[]]`? Seems meaningless on VS' triple-slash comments.

Reply:
It's necessary for invalid XML eg `<xref:System.Buffers.OperationStatus.Done?displayProperty=nameWithType>` which is unclosed. Perhaps we could close those tags so it could be removed. In this block they aren't present and it seems like valid XML.

I believe CDATA is also used if you want to preserve line breaks -- in our case if we're going to break these lines so they have some reasonable column width, we probably specifically don't want to preserve those line breaks. I'm not sure how this will work.

Reply:
See my other [comment](https://github.com/dotnet/runtime/pull/46716/files#r554242905).